### ISSUE TITLE
fake_useragent.errors.FakeUserAgentError: Maximum amount of retries reached

### ISSUE THREAD

**Issue by rootkit7628** [Abdur-rahmaanJ/download-stats#1]
I was testing the cli version with my package `ampalibe` and with `--self` options too but I'm still getting the following error
```py
Stats for download-stats
Error occurred during loading data. Trying to use cache server https://fake-useragent.herokuapp.com/browsers/0.1.11
Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1276, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1322, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1271, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1031, in _send_output
    self.send(msg)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 969, in send
    self.connect()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1448, in connect
    self.sock = self._context.wrap_socket(self.sock,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 512, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1070, in _create
    self.do_handshake()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1341, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 64, in get
    with contextlib.closing(urlopen(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 154, in load
    for item in get_browsers(verify_ssl=verify_ssl):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 97, in get_browsers
    html = get(settings.BROWSERS_STATS_PAGE, verify_ssl=verify_ssl)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 84, in get
    raise FakeUserAgentError('Maximum amount of retries reached')
fake_useragent.errors.FakeUserAgentError: Maximum amount of retries reached
Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1276, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1322, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1271, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1031, in _send_output
    self.send(msg)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 969, in send
    self.connect()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1448, in connect
    self.sock = self._context.wrap_socket(self.sock,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 512, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1070, in _create
    self.do_handshake()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1341, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 64, in get
    with contextlib.closing(urlopen(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 154, in load
    for item in get_browsers(verify_ssl=verify_ssl):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 97, in get_browsers
    html = get(settings.BROWSERS_STATS_PAGE, verify_ssl=verify_ssl)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 84, in get
    raise FakeUserAgentError('Maximum amount of retries reached')
fake_useragent.errors.FakeUserAgentError: Maximum amount of retries reached

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1276, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1322, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1271, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1031, in _send_output
    self.send(msg)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 969, in send
    self.connect()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/http/client.py", line 1448, in connect
    self.sock = self._context.wrap_socket(self.sock,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 512, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1070, in _create
    self.do_handshake()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/ssl.py", line 1341, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 64, in get
    with contextlib.closing(urlopen(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/urllib/request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rxxt/.pyenv/versions/3.10.0/bin/download-stats", line 8, in <module>
    sys.exit(main())
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/download_stats/main.py", line 83, in main
    summary_table('download-stats')
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/download_stats/main.py", line 14, in summary_table
    data = stats(package)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/download_stats/pepy.py", line 27, in stats
    ua = UserAgent()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/fake.py", line 69, in __init__
    self.load()
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/fake.py", line 75, in load
    self.data = load_cached(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 250, in load_cached
    update(path, use_cache_server=use_cache_server, verify_ssl=verify_ssl)
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 245, in update
    write(path, load(use_cache_server=use_cache_server, verify_ssl=verify_ssl))
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 187, in load
    ret = json.loads(get(
  File "/home/rxxt/.pyenv/versions/3.10.0/lib/python3.10/site-packages/fake_useragent/utils.py", line 84, in get
    raise FakeUserAgentError('Maximum amount of retries reached')
fake_useragent.errors.FakeUserAgentError: Maximum amount of retries reached
```

**Comment by Abdur-rahmaanJ** [comment]
You got the error at the first time or after fetching stats multiple times?

**Comment by rootkit7628** [comment]
at the first time! I've just installed the package and on the first try, I got this error.

**Comment by kherin** [comment]
Facing same issue also upon first installation.

Context:
- Platform: MacOS - Terminal
- Connection: No VPN
- Python Version: 3.8.1
- Command: `download-stats --self`

**Comment by Abdur-rahmaanJ** [comment]
@rootkit7628 What operating system are you using?

**Comment by Abdur-rahmaanJ** [comment]
@kherin Try using Python 3.10

**Comment by kherin** [comment]
> @kherin Try using Python 3.10

Yup working now. Thanks 🙏

**Comment by rootkit7628** [comment]
> @rootkit7628 What operating system are you using?

I'm using `Ubuntu 22.04` and `python 3.10`

**Comment by Abdur-rahmaanJ** [comment]
@kherin and @rootkit7628 Should be fixed in [1.2.1](https://github.com/Abdur-rahmaanJ/download-stats/releases/tag/v1.2.1)
